frontend/src/components/limited-textarea/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { Text, Textarea, TextareaProps } from "theme-ui";
+
+type LimitedTextareaPros = TextareaProps & {
+  limit: number;
+};
+const LimitedTextarea = ({
```

Comment:
we usually prefer export naming constants

```suggestion
export const LimitedTextarea = ({
```